website/docs/reference/widgets/select.md

```diff
@@ -16,6 +16,11 @@ A Dropdown **Options** can be populated from a data source like an API / Query b
 }}
 ```
 
+To access object values, you should use the something like `obj["test"]` instead of `obj.test`. For example:
```

Comment:
```suggestion
To access the values of an object with attribute names that contain spaces, use obj["<ATTRIBUTE_NAME"] instead of `obj.<ATTRIBUTE_NAME>`, where "<ATTRIBUTE_NAME>" is the placeholder for an attribute name.  For example, the attribute is "income tax".
```